sound/soc/sof/pm.c

```diff
@@ -164,7 +164,7 @@ static int sof_resume(struct device *dev, bool runtime_resume)
 		ret = tplg_ops->set_up_all_pipelines(sdev, false);
```

Comment:
nit for metrics on our side: Reported-by: Curtis Malainey <cujomalainey@chromium.org>

Reply:
missed Curtis' Reported-by in the update